### PR TITLE
Jason.hunsberger/content-pack-update-20240620

### PR DESCRIPTION
**Pull request by jnhunsberger**
### What does this PR do? What is the motivation?

fixing a mislabelled menu item and adding in missing documentation links

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_siem/content_packs.md [hunk 1]
```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
...
+
+Collect and monitor logs from Cisco Umbrella to gain insights into DNS and Proxy logs
+
+The [Cisco Umbrella Content Pack][39] includes:
+- [Detection Rules][40]
+- An interactive dashboard
+
+## Web Security Content Packs
+
+### nginx
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:181]:
```suggestion
### Nginx
```

File: content/en/security/cloud_siem/content_packs.md [hunk 2]
```diff
@@ -25,9 +25,10 @@ further_reading:
 
 {{< whatsnext desc="Content Packs are grouped into the following categories:" >}}
     {{< nextlink href="/security/cloud_siem/content_packs#cloud-audit-content-packs" >}}<u>Cloud Audit</u>: AWS CloudTrail, Azure Security, GCP Audit Logs, Kubernetes Audit Logs{{< /nextlink >}}
-    {{< nextlink href="/security/cloud_siem/content_packs#authentication-content-packs" >}}<u>Authentication</u>: 1Password, Auth0, JumpCloud, Okta{{< /nextlink >}}
-    {{< nextlink href="/security/cloud_siem/content_packs#collaboration-content-packs" >}}<u>Collaboration</u>: Google Workspace, Microsoft 365{{< /nextlink >}}
-    {{< nextlink href="/security/cloud_siem/content_packs#network-content-packs" >}}<u>Network</u>: Cloudflare{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#authentication-content-packs" >}}<u>Authentication</u>: 1Password, Auth0, Cisco DUO, JumpCloud, Okta{{< /nextlink >}}
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:28]:
```suggestion
    {{< nextlink href="/security/cloud_siem/content_packs#authentication-content-packs" >}}<u>Authentication</u>: 1Password, Auth0, Cisco Duo, JumpCloud, Okta{{< /nextlink >}}
```

File: content/en/security/cloud_siem/content_packs.md [hunk 3]
```diff
@@ -90,6 +91,14 @@ The [Auth0 Content Pack][15] includes:
 - [Detection Rules][16]
 - An interactive dashboard
 
+### Cisco DUO
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:94]:
```suggestion
### Cisco Duo
```

File: content/en/security/cloud_siem/content_packs.md [hunk 4]
```diff
@@ -90,6 +91,14 @@ The [Auth0 Content Pack][15] includes:
...
+
+Monitor and analyze MFA and secure access logs from Cisco DUO.
+
+The [Cisco DUO Content Pack][31] includes:
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:98]:
```suggestion
The [Cisco Duo Content Pack][31] includes:
```

File: content/en/security/cloud_siem/content_packs.md [hunk 5]
```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
...
+- [Detection Rules][40]
+- An interactive dashboard
+
+## Web Security Content Packs
+
+### nginx
+
+Monitor and respond to web-based risks with Nginx
+
+The [nginx Content Pack][41] includes:
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:185]:
```suggestion
The [Nginx Content Pack][41] includes:
```

File: content/en/security/cloud_siem/content_packs.md [hunk 6]
```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
...
+
+Analyze traffic and detect threats with Palo Alto Networks Firewall.
+
+The [Palo Alto Networks Firewall Content Pack][37] includes:
+- [Detection Rules][38]
+- An interactive dashboard
+
+### Cisco Umbrella
+
+Collect and monitor logs from Cisco Umbrella to gain insights into DNS and Proxy logs
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:173]:
```suggestion
Collect and monitor logs from Cisco Umbrella to gain insights into DNS and Proxy logs.
```

File: content/en/security/cloud_siem/content_packs.md [hunk 7]
```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
...
+
+The [Cisco Umbrella Content Pack][39] includes:
+- [Detection Rules][40]
+- An interactive dashboard
+
+## Web Security Content Packs
+
+### nginx
+
+Monitor and respond to web-based risks with Nginx
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:183]:
```suggestion
Monitor and respond to web-based risks with Nginx.
```

File: content/en/security/cloud_siem/content_packs.md [hunk 8]
```diff
@@ -90,6 +91,14 @@ The [Auth0 Content Pack][15] includes:
 - [Detection Rules][16]
 - An interactive dashboard
 
+### Cisco DUO
+
+Monitor and analyze MFA and secure access logs from Cisco DUO.
```
Comment (alai97) [inline content/en/security/cloud_siem/content_packs.md:96]:
```suggestion
Monitor and analyze MFA and secure access logs from Cisco Duo.
```

**Review by alai97 — COMMENTED** [review]
Looks good, small suggestions.

File: content/en/security/cloud_siem/content_packs.md (hunk 2 above)
Comment (jnhunsberger) [reply content/en/security/cloud_siem/content_packs.md:28]: Cisco has taken multiple approaches to the capitalization of their DUO's name. We have tried to be consistent and use `DUO` across the product including in our UI. We would like to keep `DUO` as is.

File: content/en/security/cloud_siem/content_packs.md (hunk 1 above)
Comment (jnhunsberger) [reply content/en/security/cloud_siem/content_packs.md:181]: The previous update had also changed the capitalization. Which way do we want it? `Nginx`, `nginx`, or `NGINX`?

Comment (jnhunsberger) [reply content/en/security/cloud_siem/content_packs.md:181]: Let's go with `NGINX`

**Review by alai97 — APPROVED** [review]

**Pull request merged**